MotChallenge2DBox
YOLOv11
Traceback (most recent call last):
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 113, in evaluate
    res[curr_seq] = eval_sequence(curr_seq, dataset, tracker, class_list, metrics_list,
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 222, in eval_sequence
    data = dataset.get_preprocessed_seq_data(raw_data, cls)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\datasets\mot_challenge_2d_box.py", line 324, in get_preprocessed_seq_data
    self._check_unique_ids(raw_data)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\datasets\_base_dataset.py", line 315, in _check_unique_ids
    raise TrackEvalException(exc_str)
TrackEval.trackeval.utils.TrackEvalException: Tracker predicts the same ID more than once in a single timestep (seq: MOT16-02, frame: 1, ids: -1)





MotChallenge2DBox
YOLOv11
Traceback (most recent call last):
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 113, in evaluate
    res[curr_seq] = eval_sequence(curr_seq, dataset, tracker, class_list, metrics_list,
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 222, in eval_sequence
    data = dataset.get_preprocessed_seq_data(raw_data, cls)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\datasets\mot_challenge_2d_box.py", line 324, in get_preprocessed_seq_data
    self._check_unique_ids(raw_data)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\datasets\_base_dataset.py", line 315, in _check_unique_ids
    raise TrackEvalException(exc_str)
TrackEval.trackeval.utils.TrackEvalException: Tracker predicts the same ID more than once in a single timestep (seq: MOT16-02, frame: 1, ids: -1)





MotChallenge2DBox
YOLOv11
Traceback (most recent call last):
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 113, in evaluate
    res[curr_seq] = eval_sequence(curr_seq, dataset, tracker, class_list, metrics_list,
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 222, in eval_sequence
    data = dataset.get_preprocessed_seq_data(raw_data, cls)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\datasets\mot_challenge_2d_box.py", line 354, in get_preprocessed_seq_data
    raise TrackEvalException(
TrackEval.trackeval.utils.TrackEvalException: Evaluation is only valid for pedestrian class. Non pedestrian class (26) found in sequence MOT16-02 at timestep 1.





MotChallenge2DBox
YOLOv11
Traceback (most recent call last):
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 113, in evaluate
    res[curr_seq] = eval_sequence(curr_seq, dataset, tracker, class_list, metrics_list,
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 222, in eval_sequence
    data = dataset.get_preprocessed_seq_data(raw_data, cls)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\datasets\mot_challenge_2d_box.py", line 354, in get_preprocessed_seq_data
    raise TrackEvalException(
TrackEval.trackeval.utils.TrackEvalException: Evaluation is only valid for pedestrian class. Non pedestrian class (26) found in sequence MOT16-02 at timestep 1.





MotChallenge2DBox
YOLOv11
Traceback (most recent call last):
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 113, in evaluate
    res[curr_seq] = eval_sequence(curr_seq, dataset, tracker, class_list, metrics_list,
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 222, in eval_sequence
    data = dataset.get_preprocessed_seq_data(raw_data, cls)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\datasets\mot_challenge_2d_box.py", line 354, in get_preprocessed_seq_data
    raise TrackEvalException(
TrackEval.trackeval.utils.TrackEvalException: Evaluation is only valid for pedestrian class. Non pedestrian class (26) found in sequence MOT16-02 at timestep 1.





MotChallenge2DBox
YOLOv11
Traceback (most recent call last):
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 113, in evaluate
    res[curr_seq] = eval_sequence(curr_seq, dataset, tracker, class_list, metrics_list,
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 222, in eval_sequence
    data = dataset.get_preprocessed_seq_data(raw_data, cls)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\datasets\mot_challenge_2d_box.py", line 354, in get_preprocessed_seq_data
    raise TrackEvalException(
TrackEval.trackeval.utils.TrackEvalException: Evaluation is only valid for pedestrian class. Non pedestrian class (26) found in sequence MOT16-02 at timestep 1.





MotChallenge2DBox
YOLOv11
Traceback (most recent call last):
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 113, in evaluate
    res[curr_seq] = eval_sequence(curr_seq, dataset, tracker, class_list, metrics_list,
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 222, in eval_sequence
    data = dataset.get_preprocessed_seq_data(raw_data, cls)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\datasets\mot_challenge_2d_box.py", line 354, in get_preprocessed_seq_data
    raise TrackEvalException(
TrackEval.trackeval.utils.TrackEvalException: Evaluation is only valid for pedestrian class. Non pedestrian class (26) found in sequence MOT16-02 at timestep 1.





MotChallenge2DBox
YOLOv11
Traceback (most recent call last):
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 113, in evaluate
    res[curr_seq] = eval_sequence(curr_seq, dataset, tracker, class_list, metrics_list,
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 222, in eval_sequence
    data = dataset.get_preprocessed_seq_data(raw_data, cls)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\datasets\mot_challenge_2d_box.py", line 354, in get_preprocessed_seq_data
    raise TrackEvalException(
TrackEval.trackeval.utils.TrackEvalException: Evaluation is only valid for pedestrian class. Non pedestrian class (26) found in sequence MOT16Lite-02 at timestep 1.





MotChallenge2DBox
YOLOv11
Traceback (most recent call last):
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 113, in evaluate
    res[curr_seq] = eval_sequence(curr_seq, dataset, tracker, class_list, metrics_list,
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 222, in eval_sequence
    data = dataset.get_preprocessed_seq_data(raw_data, cls)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\datasets\mot_challenge_2d_box.py", line 324, in get_preprocessed_seq_data
    self._check_unique_ids(raw_data)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\datasets\_base_dataset.py", line 315, in _check_unique_ids
    raise TrackEvalException(exc_str)
TrackEval.trackeval.utils.TrackEvalException: Tracker predicts the same ID more than once in a single timestep (seq: MOT16Lite-02, frame: 1, ids: -1)





MotChallenge2DBox
YOLOv11
Traceback (most recent call last):
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 113, in evaluate
    res[curr_seq] = eval_sequence(curr_seq, dataset, tracker, class_list, metrics_list,
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 222, in eval_sequence
    data = dataset.get_preprocessed_seq_data(raw_data, cls)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\datasets\mot_challenge_2d_box.py", line 324, in get_preprocessed_seq_data
    self._check_unique_ids(raw_data)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\datasets\_base_dataset.py", line 315, in _check_unique_ids
    raise TrackEvalException(exc_str)
TrackEval.trackeval.utils.TrackEvalException: Tracker predicts the same ID more than once in a single timestep (seq: MOT16Lite-02, frame: 1, ids: -1)





MotChallenge2DBox
YOLOv11
Traceback (most recent call last):
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 113, in evaluate
    res[curr_seq] = eval_sequence(curr_seq, dataset, tracker, class_list, metrics_list,
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\eval.py", line 224, in eval_sequence
    seq_res[cls][met_name] = metric.eval_sequence(raw_data)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\_timing.py", line 17, in wrap
    result = f(*args, **kw)
  File "C:\Documents\Projects-Code\ObjectDetectionExperement\YOLOv11\TrackEval\trackeval\metrics\hota.py", line 36, in eval_sequence
    if data['num_tracker_dets'] == 0:
KeyError: 'num_tracker_dets'





